File: algorithms/Sorting/Easy/SimpleSort/VSDSimpleSort.cpp


#include<iostream>

using namespace std;

void swap(int arr[],int i,int j){
    int temp=arr[i];
    arr[i]=arr[j];
    arr[j]=temp;
}

void VSDSimpleSort(int arr[],int n){
    for(int i=0;i<n;i++){
        for(int j=0;j<n;j++){
            if(arr[i]<arr[j]){
                swap(arr,i,j);
            }
        }
    }
    for(int i=0;i<n;i++){
        cout<<arr[i]<<" ";
    }
    cout<<endl;
}


int main(){
    int n=7;
    int arr[]={2,24,12,96, 456, 899,34};
    VSDSimpleSort(arr,n);
    return 0;
}

File: algorithms/Sorting/Easy/SimpleSort/VSDSimpleSort.java
/*Copyrights to vsdevelopers.io*/
/*For more programs visit vsdevelopers.io */
/*Java program for Simple sorting*/

public class Main {
	
	//Function to perform simple sorting
	public static void VSDsort(int n,int[] a) {
		for(int i=0;i<n;i++) {
			for(int j=0;j<n;j++) {
				if (a[i] < a[j]) 
					VSDswapElementsAt(a,i,j);
			}
		}
	}
	//Function to swap the elements of array
	public static void VSDswapElementsAt(int[]a,int i,int j) {
			int temp=a[i];
	     	a[i]=a[j];
	     	a[j]=temp;
	}
	
	
	//Function to print output
	public static void VSDprintOutput(int n,int a[]) {
		System.out.println("The sorted order is:");
		for(int i=0;i<n;i++) {
			System.out.println(a[i]);
		}
	}
	public static void main(String args[]) {
		int[] a={1, 2, 53, 12, 45, 11, 56};
		int n=a.length;
		VSDsort(n,a);//Call to sort the array
		VSDprintOutput(n,a);//Printing output
	}
}
